lib/src/main/java/com/auth0/jwt/impl/JsonNodeClaim.java

```diff
@@ -91,6 +92,18 @@ public Date asDate() {
     }
 
     @Override
+    public Map<String, Object> asMap() throws JWTDecodeException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
not really this PR, but why isn't there just one `ObjectMapper` (even static) instead of creating a new one every time?
the same can be said about the `TypeReference`